gradle.properties

```diff
@@ -19,7 +19,7 @@
 #
 
 rootProject.name=fdb-record-layer
-version=3.0
+version=3.1
```

Comment:
A good start, but I believe there are a few more things that need to be done as part of the version bump:

1. ReleaseNotes.md needs to be updated so that the description of what changed in 3.0 is below the "next_release" block. It would also be good to add a 3.1 header and a description of what's new in 3.1 (which is fine to be just the Java 11 changes)
2. One of our pages in the docs has the current version in it just for reference. That one is less important to update, but probably good practice while we're there.

I believe that you should be able to model the changes after the changes in this commit (bumping to 3.0): https://github.com/FoundationDB/fdb-record-layer/commit/9a80491bfe43e870a31dd17a5f46b31d55a12672

Reply:
Done.